CONTRIBUTING.md

```diff
@@ -0,0 +1,96 @@
+* The concepts (e.g., `/content/en/docs/concepts/`): a collection of documents each of which explains about a specific concept in KubeFleet;
+* The getting started tutorials (e.g., `/content/en/docs/getting-started`): a collection of tutorials each of which help developers to get familiar with KubeFleet with a specific setup;
+* the how-to guides (e.g., `/content/en/docs/how-tos`): a collection of guides each of which explains about how to use a specific KubeFleet feature;
+* the FAQ (e.g., `/content/en/docs/faq`), which answers some of the most common questions developers might have with KubeFleet;
+* the KubeFleet API references (e.g., `/content/en/docs/api-reference`).
+
+All the documents includes are written in [the Markdown language](https://en.wikipedia.org/wiki/Markdown). Both the basic syntax and the extended syntax are supported. To edit the documentation, simply find the Markdown file of interest, make some changes, and submit them as a PR.
+
+> The KubeFleet API references are auto-generated. Do not edit it manually.
+
+If you would like to add a new document to one of the collections (concepts, getting started tutorials, or how-to guides), create a Markdown file in the corresponding subdirectory, and submit it in a PR. The Markdown file should start with a header as follows:
+
+```
+---
+title: YOUR-TITLE
+description: YOUR-DOCUMENT-DESCRIPTION
+weight: YOUR-PAGE-WEIGHT
+---
+```
+
+Replace `YOUR-TITLE` and `YOUR-DOCUMENT-DESCRIPTION` with values of your own respectively; these values will be used when rendering the Markdown file into a webpage. Assign a proper numeric value to `YOUR-PAGE-WEIGHT`, which determines the order of your document in the table of contents: for example, the document with the least weight in the concepts collection of documents would appear at the top of the TOC for the concepts section.
+
+## Building and Deploying the Website
+
+The KubeFleet repository has set up a GitHub Actions pipeline to build the website and deploy it to [GitHub Pages](https://kubefleet-dev.github.io/website/) every time a PR is merged into the `main` branch. Track the build + deployment status in the [GitHub Workflow page](https://github.com/kubefleet-dev/website/actions/workflows/github-pages.yml); once deployed, your changes should appear on the [GitHub Pages](https://kubefleet-dev.github.io/website/) right away.
+
+The same pipeline would also run on each PR; the workflow would attempt to build the website, but no deployment is performed. Use this check to verify if your changes are valid.
+
+### Building and Deploying the Website Locally
+
+Often contributors would need to build and deploy the website locally to check their progress. To do so,
+
+* Install [Hugo](https://gohugo.io).
+    * Hugo is the static site generator KubeFleet uses to generate the website. It is recommended that you install the extended version.
+* Clone the repository.
```

Comment:
which repo? This repo?

Reply:
Hi Ryan! Yeah, it is this repository. I have tweaked the wording to make it more clear.